frontend/views/components/modal/ModalClose.vue

```diff
@@ -1,7 +1,7 @@
 <template lang='pug'>
   button.c-modal-close(
     :class='{ backOnMobile, fullscreen }'
-    @click.self='$emit("close")'
+    @click.self.stop='$emit("close")'
```

Comment:
@SebinSong just curious if you remember what this change was for?

Reply:
@taoeffect I can't exactly remember but I think I was fixing one of the bugs Alex raised during his review. it only prevents the `click` event from the `X` button in modal from bubbling up, so I don't think the change will break something anywhere else in the app but if an issue is encountered regarding this later on, I will remove it.